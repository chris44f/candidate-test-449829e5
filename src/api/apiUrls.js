const apiUrls = {
  CHARACTERS: '/characters.json',
};

export default apiUrls;
