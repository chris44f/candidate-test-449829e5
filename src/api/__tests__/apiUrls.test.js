import * as urls from '../apiUrls';

describe('urls', () => {
  it('remains unchanged', () => {
    expect(urls).toMatchSnapshot();
  });
});
