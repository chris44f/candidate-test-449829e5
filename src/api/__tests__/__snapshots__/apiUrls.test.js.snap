// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`urls remains unchanged 1`] = `
Object {
  "default": Object {
    "CHARACTERS": "/characters.json",
  },
}
`;
